README.md

```
@@ -36,8 +35,16 @@
 
 ## 🕶 Try it yourself!
 
+#### Ask any questions
+
 https://github.com/user-attachments/assets/e4e78006-0088-4094-b58f-8868689bf787
 
+#### GenBI
+
+https://github.com/user-attachments/assets/90ad1d35-bb1e-490b-9676-b29863ff090b
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)